.github/workflows/cross.yml

```diff
@@ -21,62 +21,67 @@ env:
     runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
         target:
-          - aarch64-linux-android
-          - aarch64-unknown-linux-gnu
-          - aarch64-unknown-linux-musl
-          - arm-linux-androideabi
-          - arm-unknown-linux-gnueabihf
-          - armv7-linux-androideabi
-          - armv7-unknown-linux-gnueabihf
-          - i686-unknown-linux-gnu
-          - powerpc-unknown-linux-gnu
-          - powerpc64-unknown-linux-gnu
-          - powerpc64le-unknown-linux-gnu
-          - riscv64gc-unknown-linux-gnu
-          - s390x-unknown-linux-gnu
-          - x86_64-pc-windows-gnu
-          - x86_64-unknown-linux-musl
+          - [aarch64-linux-android, 0]
+          - [aarch64-unknown-linux-gnu, 1]
+          - [aarch64-unknown-linux-musl, 1]
+          - [arm-linux-androideabi, 0]
+          - [arm-unknown-linux-gnueabihf, 0]
+          - [arm-unknown-linux-musleabi, 1]
+          - [arm-unknown-linux-musleabihf, 1]
+          - [armv7-linux-androideabi, 0]
+          - [armv7-unknown-linux-gnueabihf, 0]
+          - [i686-unknown-linux-gnu, 0]
+          - [powerpc-unknown-linux-gnu, 1]
+          - [powerpc64-unknown-linux-gnu, 1]
+          - [powerpc64le-unknown-linux-gnu, 1]
+          - [riscv64gc-unknown-linux-gnu, 0]
+          - [s390x-unknown-linux-gnu, 0]
+          - [x86_64-pc-windows-gnu, 0]
+          - [x86_64-unknown-linux-musl, 0]
     steps:
       - uses: actions/checkout@v3
         with:
-          submodules: 'recursive'
+          submodules: "recursive"
       - uses: dtolnay/rust-toolchain@master
         with:
-          toolchain: 'stable'
+          toolchain: "stable"
       - name: Install cross
         run: cargo install cross --git https://github.com/cross-rs/cross
       - uses: dtolnay/rust-toolchain@master
         id: toolchain
         with:
-          toolchain: 'stable'
-          target: ${{ matrix.target }}
+          toolchain: "stable"
+          target: ${{ matrix.target[0] }}
       - name: Set Rust toolchain override
         run: rustup override set ${{ steps.toolchain.outputs.name }}
       # The flag below is set to avoid the following error with GCC 11.4.0 on the riscv64 platform:
       #   /home/runner/work/aws-lc-rs/aws-lc-rs/aws-lc-sys/aws-lc/crypto/pem/pem_lib.c:707:11: error: 'strncmp' of strings of length 1 and 9 and bound of 9 evaluates to nonzero [-Werror=string-compare]
       #    707 |       if (strncmp(buf, "-----END ", 9) == 0) {
       #        |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
-      - if: ${{ matrix.target == 'riscv64gc-unknown-linux-gnu' }}
+      - if: ${{ matrix.target[0] == 'riscv64gc-unknown-linux-gnu' }}
         run: |
           echo 'AWS_LC_SYS_CFLAGS="-Wno-string-compare"' >> "$GITHUB_ENV"
-      - name: Cross-compilation (build debug)
-        run: cross build -p aws-lc-rs --features unstable --target ${{ matrix.target }}
+      - if: ${{ !startsWith(matrix.target[0], 'x86_64') }}
```

Comment:
1. why just `x86_64`?
2. doesn't this mean we lose cross-compilation coverage for `x86_64` targets?

Reply:
* This disables the "slow" tests only for platforms that are emulated (i.e., the non-`x86_64` targets). The `x86_64` will continue to run all tests.
* Although we are disabling several slow-running tests, this change should have very minimal effect on overall code coverage. For example, we still test all RSA functionality with 2048-bit keys, and only disable RSA tests that require the generation of keys sized 3072-bit or larger.